src/libs/ReportUtils.js

```diff
@@ -2398,6 +2399,16 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, iouRep
         return false;
     }
 
+    // Hide thread reports that haven't been commented on
+    if (isThread(report) && _.isEmpty(report.lastMessageText)) {
```

Comment:
I think it's better to use [getLastVisibleMessage](https://github.com/Expensify/App/blob/65ee88e55afcdd0f60db506c95b20e6ccb2139f9/src/libs/ReportActionsUtils.js#L384), since it'll loop over the report actions, in case the report object missed an update. We should also check for the draft comment here

```suggestion
    if (isThread(report) && _.isEmpty(ReportActionsUtils.getLastVisibleMessage(report)['lastMessageText']) && report.hasDraftComment) {
```

Reply:
Makes sense! Thks for the suggestion and already putting the `lastMessageText` check in the snippet since that function can also return the created action : )